domains/drywall.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ThatWirdGuy",
+           "discord": "1304861678483275798",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.SKUDN9_wA5Eog4MI924jL6hjeVp0Pv527mYr_ctrcGTO2DVtduC9HDbRXQP_2DQgiD9x2QL9BwHmc3tPgNr2sGWeZO3y8s2mpZaBBYG9H-2o2PaBc6D6fRRKvc3SduCJTIVUHTKBsMkx6392dZWlJJu3hZwrWF5nYPfonETbBHLnh9nPFEg0rYNXzqPWBkEoSGRysSMJNRF-0r5pnxu_3vMAdqJWjgnMrV0V8eEO6DZ3BdB4KTni8_PbXVdPjyUFcLcJLKOWLhhHgibdmKGwdsfTNhjYbnn9u84TGAY7XO8pZ-jweChhMvkpePZthczV-2cWrd-f8xnOpKN_NoZDww.ZGEcFLphndqfj_j37-5aZg.EDVBKKfwenHBKDGcB-NcDXgQ259X-A1sqfajr6533EgefmwcLxJpYEAV2FUyWocv8b1JAmJaboJwnrITQvHWXs-2fv5QQugdBR7TA8km76Q.g7JcHsNE9xGAGqTkAqRZBw"
+        },
+    
+        "record": {
+            "CNAME": "hosting by is-a.dev"
```

Comment:
Invalid CNAME record, should be a valid hostname. See docs for information: https://docs.is-a.dev